README.md

```diff
@@ -6,6 +6,25 @@ Set `GOMAXPROCS` to match container CPU quota.
 
 `go get -u go.uber.org/automaxprocs`
 
+## Quick Start
+
+```go
+import (
+  "log"
+
+  "go.uber.org/automaxprocs"
```

Comment:
Should we break up the package into 2 parts:
 - `automaxprocs/maxprocs`: With an API that allows for explicit set
 - `automaxprocs` - blank importable with side effects

In terms of usage, it's much simpler to do a blank import. `MAXPROCS` is a global value, so I'm not against the global nature of the current package design.